src/runtime/typemanager.cs

```diff
@@ -280,6 +280,7 @@ internal static IntPtr CreateType(ManagedType impl, Type clrType)
             {
                 throw new PythonException();
             }
+            OperatorMethod.FixupSlots(type, clrType);
```

Comment:
Please address the comment that was made on this file in PR #907

Reply:
I have moved it up before pytype_ready